internal/config/template.go

```diff
@@ -0,0 +1,18 @@
+package config
+
+import (
+	"html/template"
+)
+
+var (
+	Tmpl *template.Template
+	err  error
+)
+
+func GetTemplates() error {
```

Comment:
seems like this could just go in the root main.go since we aren't gonna test it!

Reply:
commented on this above, but in short I don't think it can go in `main.go` since it needs to be exported and used in other packages